### PR TITLE
Tech-416- how to create a validator node

### PR DESCRIPTION
**Pull request by mo-c4t**
- note for the version compatible with testnet is v0.2.1-rc2
- Setup camino node manually: replace github ssh url to https url
- Setup camino node manually: adding a hint about public-ip option
- steps to add a validator node
- setup camino node from install script: note to add network-id option in the config file when running on testnet
- replacing "an Camino node" to "a Camino node"?
- adding Ubuntu 22.04 to the supported OS

### CODE REVIEW

**Review by c4t-ag — CHANGES_REQUESTED** [review]
This will help! Thx much!
In the initial article there is often person switching between neutral / you / their ....
I suggest that some native nglish speaker jumps in here

**Comment by mo-c4t** [comment]
> This will help! Thx much! In the initial article there is often person switching between neutral / you / their .... I suggest that some native nglish speaker jumps in here

Do you mean this sentence 
"Validators must know their public facing IP addresses so they can let other nodes know how to connect to them. So If your node will run as a validator, you need to use `--public-ip` option. For more info check [public-ip flag](camino-node-config-flags.md#public-ip)"

or are you talking about the whole documentation ?

Who can help with this ?

File: docs/apps/nodes/set-up-node-with-installer.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Look for line that doesn't have `grep` on it. In this example, that is the secon
 
 #### Node working files
 
-If you previously ran an Camino-Node node on this computer, you will have local node files stored in `$HOME/.caminogo` directory. Those files will not be disturbed, and node set up by the script will continue operation with the same identity and state it had before. That being said, for your node's security, back up `staker.crt` and `staker.key` files, found in `$HOME/.caminogo/staking` and store them somewhere secure. You can use those files to recreate your node on a different computer if you ever need to.
+If you previously ran a Camino-Node node on this computer, you will have local node files stored in `$HOME/.caminogo` directory. Those files will not be disturbed, and node set up by the script will continue operation with the same identity and state it had before. That being said, for your node's security, back up `staker.crt` and `staker.key` files, found in `$HOME/.caminogo/staking` and store them somewhere secure. You can use those files to recreate your node on a different computer if you ever need to.
```
Comment (tabeaarnold) [inline docs/apps/nodes/set-up-node-with-installer.md:50]: If you previously ran a Camino-Node node on this computer, you will have _the_ local node files stored in `$HOME/.caminogo` directory. Those files will not be disturbed and _the_ node set up by the script will continue _its_ operation with the same identity and state it had before. That being said, for your node's security, back up `staker.crt` and `staker.key` files, found in `$HOME/.caminogo/staking` and store them somewhere secure. You can use those files to recreate your node on a different computer if you ever need to.

File: docs/apps/nodes/set-up-node-with-docker.md [hunk 2]
```diff
@@ -126,7 +126,7 @@ Note that configuration file needs to be a properly formatted `JSON` file, so sw
 
 ## What next?
 
-That's it, you're running an Camino-Node node on docker! Congratulations! Let us know you did it on our [Twitter](https://twitter.com/CaminoFndtn) or [Discord](https://discord.gg/K5THjAweFB)!
+That's it, you're running a Camino-Node node on docker! Congratulations! Let us know you did it on our [Twitter](https://twitter.com/CaminoFndtn) or [Discord](https://discord.gg/K5THjAweFB)!
```
Comment (tabeaarnold) [inline docs/apps/nodes/set-up-node-with-docker.md:129]: Share it on our

File: docs/apps/nodes/run-camino-node.md [hunk 3]
```diff
@@ -127,6 +127,8 @@ To be able to make API calls to your node from other machines, when starting up
 
 When the mainnet (Camino) goes live, you can either omit `--network-id=columbus` parameter, or pass `--network-id=camino` which is the name of our mainnet.
 
+Validators must know their public facing IP addresses so they can let other nodes know how to connect to them. So If your node will run as a validator, you need to use `--public-ip` option. For more info check [public-ip flag](camino-node-config-flags.md#public-ip)
```
Comment (tabeaarnold) [inline docs/apps/nodes/run-camino-node.md:130]: ... you need to use the ....

File: docs/apps/nodes/run-camino-node.md [hunk 4]
```diff
@@ -21,11 +21,11 @@ Camino is a lightweight protocol which let nodes run on commodity hardware. Note
...
 - RAM: 16 GiB
 - Storage: 512 GiB
-- OS: Ubuntu 18.04/20.04 or MacOS &gt;= Catalina
+- OS: Ubuntu 18.04/20.04/22.04 or MacOS &gt;= Catalina
 
-## Run an Camino Node and Send Funds
+## Run a Camino Node and Send Funds
 
-Let’s install Camino-Node, the Go implementation of an Camino node, and connect to the Camino Public Testnet (Columbus).
+Let’s install Camino-Node, the Go implementation of a Camino node, and connect to the Camino Public Testnet (Columbus).
```
Comment (tabeaarnold) [inline docs/apps/nodes/run-camino-node.md:28]: Let's install the Camino-Node

**Review by c4t-ag — APPROVED** [review]

**Pull request merged**